src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetDAO.java

```diff
@@ -23,14 +27,16 @@
 import static no.nav.pto.veilarbportefolje.util.DbUtils.parse0OR1;
 
 @Slf4j
+@Repository
 public class AktivitetDAO {
 
     private static final String AKTIVITETER = "AKTIVITETER";
     private static final String AKTIVITETID = "AKTIVITETID";
 
-    private JdbcTemplate db;
-    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+    private final JdbcTemplate db;
```

Comment:
is there any reason why to have db and namedParameterJdbcTemplate, are they pointing to the same db?